### PR TITLE
Add EFCore test project

### PR DESCRIPTION
**Pull request by AlekseyMartynov**
Okay, @kaatula this time we do it right

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@ test_script:
     -output:coverage_netcore.xml
 
   - >-
+    OpenCover\tools\OpenCover.Console -returntargetcode -register:user -oldstyle
```
Comment (kaatula) [inline appveyor.yml]: That is a brand new appveyor, how do you the ```oldstyle```??

File: net/DevExtreme.AspNet.Data.Tests.EFCore/SampleLoadOptions.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DevExtreme.AspNet.Data.Tests.EFCore {
+
+    class SampleLoadOptions : DataSourceLoadOptionsBase {
```
Comment (kaatula) [inline net/DevExtreme.AspNet.Data.Tests.EFCore/SampleLoadOptions.cs]: Not sure I got it...

File: net/DevExtreme.AspNet.Data/AssemblyInfo.cs [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 #if DEBUG
 [assembly: InternalsVisibleTo("DevExtreme.AspNet.Data.Tests")]
+[assembly: InternalsVisibleTo("DevExtreme.AspNet.Data.Tests.EF6")]
```
Comment (kaatula) [inline net/DevExtreme.AspNet.Data/AssemblyInfo.cs]: ```InternalsVisibleTo``` looks like a bad sign, doesn't it?

Comment (AlekseyMartynov) [inline net/DevExtreme.AspNet.Data/AssemblyInfo.cs]: No, [not at all](https://stackoverflow.com/q/7580710).

File: net/DevExtreme.AspNet.Data.Tests.EFCore/SampleLoadOptions.cs (hunk 2 above)
Comment (AlekseyMartynov) [inline net/DevExtreme.AspNet.Data.Tests.EFCore/SampleLoadOptions.cs]: `DataSourceLoadOptionsBase` is abstract.

File: appveyor.yml (hunk 1 above)
Comment (AlekseyMartynov) [inline appveyor.yml]: Ha ha!
Check https://github.com/OpenCover/opencover/issues/595#issuecomment-223861147

File: net/DevExtreme.AspNet.Data/AssemblyInfo.cs (hunk 3 above)
Comment (kaatula) [inline net/DevExtreme.AspNet.Data/AssemblyInfo.cs]: Thank you, I downvoted :(

File: appveyor.yml (hunk 1 above)
Comment (kaatula) [inline appveyor.yml]: 🙈

File: net/DevExtreme.AspNet.Data.Tests.EFCore/SampleLoadOptions.cs (hunk 2 above)
Comment (kaatula) [inline net/DevExtreme.AspNet.Data.Tests.EFCore/SampleLoadOptions.cs]: Yes, but why is it abstract?
Just to have a excuse to create such strange-looking classes?

Comment (AlekseyMartynov) [inline net/DevExtreme.AspNet.Data.Tests.EFCore/SampleLoadOptions.cs]: 🤡

**Pull request merged**